cmd/http/server.go

```diff
@@ -25,6 +25,8 @@ import (
 	"time"
 
 	humanize "github.com/dustin/go-humanize"
+	"golang.org/x/net/http2"
```

Comment:
looks like `net/http` package automatically uses `golang.org/x/net/http2` indirectly from go1.6

Reply:
Oh sorry.. i see that anyhow we have to directly import the latest "golang.org/x/net/http2" to use `ConfigureServer` function

Reply:
Correct, to use the newer low level features we need to use net/http2